tsconfig.eslint.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "./tsconfig.json",
-  "include": ["src/**/*.ts", "e2e/**/*", "./*.js"],
+  "include": ["src/**/*.ts", "e2e/**/*.ts"],
```

Comment:
The `./*.js` seemed to be pulling in way too much for linting, but I'll try re-enabling it.